hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ActionStatusUpdateHandlerServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package org.eclipse.hawkbit.repository.jpa;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
Please use assertions provided by AssertJ in favour of JUnit assertions

Reply:
Modified to AssertJ